1_fetch/src/my_happy_script.R

```diff
@@ -20,16 +19,18 @@ library(stringr)
 library(sbtools)
 library(whisker)
 
+# create new output directory
 project_output_dir <- 'my_dir'   # 1_fetch/out/my_dir
-
 dir.create(project_output_dir)
```

Comment:
maybe move this to the instructions section as well since it's not part of the functinos.